tsl/test/sql/vector_agg_functions.sql

```diff
@@ -101,6 +101,9 @@ set timescaledb.debug_require_vector_agg = :'guc_value';
 --set timescaledb.enable_chunkwise_aggregation to off; set timescaledb.enable_vectorized_aggregation to off; set timescaledb.debug_require_vector_agg = 'forbid';
 
 set max_parallel_workers_per_gather = 0;
+-- Disable sorting to force vectorized agg plans for min and max,
```

Comment:
This probably does nothing atm -- I removed the plans for min/max w/o grouping on main, and with grouping by segmentby the initplan is not possible. It still tests the same code though. If you checked that this disables the initplan, maybe we should return the min/max w/o grouping.

Reply:
Without disabling sort you get the init plan. I checked.

Reply:
For which query? It can't do this thing for queries like `select ss, min(cfloat4) from aggfns where cfloat8 < 1000 group by ss order by min(cfloat4), ss limit 10;` that are supposed to use hash grouping, right?

Reply:
Here's the plan and query:

![image](https://github.com/user-attachments/assets/8d3dcc05-12b0-4d81-ac9b-c1d3eb64d8c2)

Reply:
Yeah, you sent me this one before, but this is without grouping. The difference in "min" is also present with grouping by "s" and "ss", and they are already using the vectorized plans -- they have a NaN result which doesn't happen with Postgres plans.

The queries like the one in the screenshot are currently not tested, I disabled them because I didn't know how to disable the initplan.

Reply:
Oh, sorry for the confusion, apparently the thing where I disable min/max w/o grouping was not merged yet :sweat_smile: So this means your cost change is fine as is.